### DISCUSSION TITLE
Module not found: Error: Can't resolve '@badeball/cypress-cucumber-preprocessor/webpack'

### DISCUSSION THREAD

**Discussion by Gayathri-Kesavakannan** [badeball/cypress-cucumber-preprocessor#1264] [General]
I am trying to implement Cucumber in Cypress Component test. with webpack , tried example https://github.com/badeball/cypress-cucumber-preprocessor/tree/master/examples/ct-webpack-react-ts

but I am getting error on loader ERROR in ./node_modules/@cypress/webpack-dev-server/dist/browser.js (./node_modules/@cypress/webpack-dev-server/dist/loader.js!./node_modules/@cypress/webpack-dev-server/dist/browser.js) Module not found: Error: Can't resolve '@badeball/cypress-cucumber-preprocessor/webpack' I have latest version as below . any idea what I am missing here

"ts-loader": "^9.5.1",
"react-scripts": "^5.0.1",
"@badeball/cypress-cucumber-preprocessor": "^21.0.3",
"@cypress/webpack-preprocessor": "^5.8.0",
"@cypress/webpack-dev-server": "^3.11.0",
getting same error for E2E with example https://github.com/badeball/cypress-cucumber-preprocessor/tree/master/examples/webpack-cjs

Please share details on webpack & cypress

Thanks

**Comment by badeball** [comment]
What node version are you using?
